components/compliance-service/reporting/relaxting/es_types.go

```diff
@@ -110,6 +111,7 @@ type ESInSpecReportControl struct {
 	Status     string                            `json:"status"`
 	Results    []*ESInSpecReportControlsResult   `json:"results"`
 	StringTags []ESInSpecReportControlStringTags `json:"string_tags"`
+	Refs       []*reportingapi.Ref               `json:"refs"`
```

Comment:
is this ok? i see the other types in here are types defined in this file. is there a reason i should avoid using reportingapi.Ref here as a type? cc @alexpop @rickmarry

Reply:
I don't see it as a problem. All attributes of Ref are exported so you won't run into any visibility issues in using this.  There are other structs in this files that use type that are defined in other packages too.

Reply:
The `reportingapi.Ref` is defined for the RPC API and the `ESInSpecReportControl` is defined for Elasticsearch. I would say that another `Ref` should be defined in this file. Maybe a "ESRef".

Reply:
>  I would say that another `Ref` should be defined in this file. Maybe a "ESRef".
I suppose for consistency you're right @lancewf but it seems like we have a lot of these mirror structs in our code

Reply:
Creating the new ESRef struct would allow flexibility in changing how the data comes in through the API and how we store the data with Elasticsearch.

Reply:
cool